test/roots.jl

```diff
@@ -66,32 +102,43 @@ end
     XX = IntervalBox(-5..6, 2)
     tol = 1e-5
 
+    # Newton
     rts = roots(gradf, XX, Newton, tol)
     @test length(rts) == 25
+    @test all_unique(rts)
     @test rts == roots(gradf, XX, Newton, tol; deriv = xx -> ForwardDiff.jacobian(gradf, xx))
+
+    # Krawczyk
```

Comment:
This part and Newton above can be done with a `for` loop instead of duplicating the code.